bundles/org.openhab.binding.emby/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,195 @@
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- Sample Thing Type -->
+	<bridge-type id="controller">
+		<label>EMBY Server</label>
+		<description>This is the Bridge to an instance of an EMBY server you want to connect to.</description>
+		<config-description>
+			<parameter name="api" type="text" required="true">
+				<label>API Key</label>
+				<description>This is the API key generated from EMBY used for Authorization.</description>
+			</parameter>
+			<parameter name="buffer_size" type="integer">
+				<label>Web Socket Buffer Size</label>
+				<description>Here you can define a custom size for the websocket buffer size. Default is 100000</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>ipAddress</label>
+				<description>This is the ip address of the EMBY Server.</description>
+			</parameter>
+			<parameter name="port" type="text" required="true">
+				<label>Port</label>
+				<description>This is the port of the EMBY server.</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" required="true">
+				<label>Refresh Parameter</label>
+				<description>This is the refresh interval in milliseconds that will be sent to the websocket.</description>
+			</parameter>
+		</config-description>
+	</bridge-type>
+	<thing-type id="device" extensible="generalCommand,generalCommand_withArguments">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="controller" />
+		</supported-bridge-type-refs>
+		<label>EMBY Binding Device</label>
+		<description>This  is a player device which connects to an EMBY server to play files that you want to montior.</description>
+		<channels>
+			<channel id="control" typeId="control" />
+			<channel id="stop" typeId="stop" />
+			<channel id="title" typeId="title" />
+			<channel id="mute" typeId="mute" />
+			<channel id="showtitle" typeId="showtitle" />
+			<channel id="imageurl" typeId="imageurl" />
+			<channel id="currenttime" typeId="currenttime" />
+			<channel id="duration" typeId="duration" />
+			<channel id="mediatype" typeId="mediatype" />
+			<channel id="sendplay" typeId="sendplay" />
+		</channels>
+		<config-description>
+			<parameter name="deviceID" type="text" required="true">
+				<label>DeviceID</label>
+				<description>This is the deviceId you want to connect to to monitor.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-type id="sendplay">
+		<item-type>String</item-type>
+		<label>Send Play</label>
+		<description>String to send a comma-delimited list of item id's that will be played by the client. The string should be in JSON format with all desired parameters from the EMBY API. https://github.com/MediaBrowser/Emby/wiki/Remote-control</description>
```

Comment:
I think this is rather an [Action](https://www.openhab.org/docs/developer/bindings/#actions-bound-to-a-thing), than a Channel.

Reply:
you are correct  will revise